backend/repository/manager.go

```diff
@@ -28,6 +28,7 @@ import (
 	"go.uber.org/thriftrw/compile"
 
 	"github.com/pkg/errors"
+	"strings"
```

Comment:
nit import order

Reply:
What does that mean?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#imports